.github/workflows/test.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
     if: (github.repository == 'kubernetes-sigs/cluster-api-provider-kubemark')
     strategy:
       matrix:
-        go-version: [1.22.x]
+        go-version: [1.21.x]
```

Comment:
Why take the go version backwards?

Reply:
i was mostly trying to sync all the versions so that everything was the same, my plan is to update to 1.22 with the cluster-api 1.8 update. so, mostly just trying to get a clean base to build on top of.